### DISCUSSION TITLE
Unit Tests for C++ Drivers

### DISCUSSION THREAD

**Discussion by benedikt-braeutigam** [tcunit/TcUnit#172] [Q&A] [answered]
Is a direct Unit Test Framework planned for C++ Drivers? 
Actual workaround, publishing methods via Sps Interface (RT Context) and then realizing unit tests.

**Comment by sagatowski** [comment]
Nothing planned in TcUnit.

A more relevant question might be; is it possible to use any of the existing C++ unit testing frameworks for C++ modules?

**Comment by densogiaichned** [comment, marked as answer]
A simple unit-test framework for TwinCAT C++: 
* https://github.com/densogiaichned/TcCppTest

**Comment by sagatowski** [comment]
@densogiaichned  Good initiative! I will add this to the F.A.Q. of TcUnit.
